stripe_test.go

```diff
@@ -496,24 +487,20 @@ func TestDo_TelemetryDisabled(t *testing.T) {
 
 // Test that telemetry metrics are sent on subsequent requests when
 // EnableTelemetry = true.
-func TestDo_TelemetryEnabled(t *testing.T) {
+func TestCall_TelemetryEnabled(t *testing.T) {
 	type testServerResponse struct {
 		APIResource
 		Message string `json:"message"`
 	}
 
-	type requestMetrics struct {
-		RequestDurationMS int    `json:"request_duration_ms"`
-		RequestID         string `json:"request_id"`
-	}
-
 	type requestTelemetry struct {
 		LastRequestMetrics requestMetrics `json:"last_request_metrics"`
 	}
 
 	message := "Hello, client."
 	requestNum := 0
 
+	var telemetry requestTelemetry
```

Comment:
nit: does this variable have to stay top level? or can it move back into the server callback?

Reply:
Will do!